lib/Doctrine/DBAL/Types/DateIntervalType.php

```diff
@@ -56,9 +58,13 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         }
 
         try {
-            return new \DateInterval($value);
+            $interval = new \DateInterval(substr($value, 1));
```

Comment:
@galeaspablo oh you are right I thought it was part of 2.5 already. Thanks for pointing that out, then we won't have a problem with BC indeeed :)